src/onnx/parse_pooling.cpp

```diff
@@ -128,14 +128,14 @@ struct parse_pooling : op_parser<parse_pooling>
             std::fill_n(values["stride"].begin(), kdims, 1);
         }
         // used to calculate the supposed output shape
-        std::vector<int64_t> orig_padding(paddings.begin(), paddings.end());
 
         std::vector<int64_t> slice_start;
         std::vector<int64_t> slice_end;
         tune_padding_size(values, paddings, count_include_pad, slice_start);
 
         if(!slice_start.empty())
         {
+            std::vector<int64_t> orig_padding(paddings.begin(), paddings.end());
```

Comment:
Looks like change is making pooling tests fail. `padding` values changes during `tune_padding_size`, `orig_padding` should take original values.